messaging/src/main/java/org/axonframework/queryhandling/QuerySubscription.java

```diff
@@ -32,7 +32,7 @@
  * @author Steven van Beelen
  * @since 3.2
  */
-class QuerySubscription<R> {
+public class QuerySubscription<R> {
```

Comment:
This would require the constructor of the `QuerySubscription` to become `public`, since it is instantiated from `SimpleQueryBus`. I'd rather keep the constructor package-private.

Reply:
Good point there, @Morlack! 
I agree it's not yet necessary to expose the construction of this class to the outside. 
I'll resolve this comment.  :+1: